src/interface/Interface.py

```diff
@@ -0,0 +1,97 @@
+#Interface
+import sys
+from Tkinter import *
+from settings_game import SettingsGame
+
+class Interface(Frame):
+    
+    def __init__(self, main=None):
+
+        """
+        main: tkinter frame object
+        constructor -- it will get Tkinter object to construct the main UI window.
+        this will call to initwindow() function , which will create and invoke all objects that will be available in the mainwindow
+        
+        """
+
+        Frame.__init__(self, main)
+        self.main = main
+        self.init_window()
+
+    def init_window(self):
+        
+        """
+        The function will contain all objects (labels, buttons, text areas and so on) to be displayed in the window and also here has defined the size of the window
+        this function will call the following command functions:
+            --settings
+            --quit
+            
+        ***** IT IS THE FIRST VERSION NEED TO IMPLEMENT THE RESTO OF THE BUTTONS AND FUNCTIONS
+        """
+
+        main.geometry("800x500+100+100")
+        main.title("Sudoku Game Interface")
+        
+        #buttons section
+        self.show_buttons()
+        
+        
+        #canvas to show the game
+        canvas_game_area = Canvas(main, height = 400, width = 400, bg = "white")
+        canvas_game_area.place(x = 150, y = 10)
+    
+        #label
+        label_time = Label(main, text = "Time: (15:25) Min.")
+        label_time.place(x = 200, y = 430)
+
+        main.mainloop()
+
+
+    def show_buttons(self):
+        """
+        funtions will show the all available buttons  in the UI
```

Comment:
misspelling funtions, besides that what are the available buttons, this is a description very generic